src/Command/UpdateCommand.php

```diff
@@ -94,7 +91,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
 
             //get versions as [version => url]
-            $versions = $info['versions'] ?: [];
```

Comment:
Why are you changing this? It doesn't seem to belong in this PR, please open a separate one if there is a need for it.